rs/nns/governance/src/governance/tests/mod.rs

```diff
@@ -133,6 +133,84 @@ mod settle_neurons_fund_participation_request_tests {
     }
 } // end mod settle_neurons_fund_participation_request_tests
 
+#[cfg(feature = "test")]
+mod settle_neurons_fund_participation_mem_tests {
```

Comment:
NCR.

I'm not entirely sure what "mem test" is supposed to mean. Here it's clearly testing the response size, not the memory size. How about `_response_size_test`?